spec/game_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "game"
+
+RSpec.describe Game do
+  it "can choose a random move" do
+    expect(["rock", "paper", "scissors"]).to include subject.random_move
+  end
+
+  describe "#result" do
+    it "is draw, when same signs" do
+      expect(subject.result("rock", "rock")).to eq "rock vs. rock: it's a draw!"
+      expect(subject.result("paper", "paper")).to eq "paper vs. paper: it's a draw!"
+      expect(subject.result("scissors", "scissors")).to eq "scissors vs. scissors: it's a draw!"
+    end
+
+    it "rock beats scissors" do
```

Comment:
For all these tests where you have it "x beats y", you could consider using a context block, and then having two its inside it for the lose and win.  It's up to you, I just think it might be easier to read, and the output for the tests would flow better.